gitman/models/config.py

```diff
@@ -373,3 +380,19 @@ def _valid_filename(filename):
     if name.startswith('.'):
         name = name[1:]
     return name in {'gitman', 'gdm'} and ext in {'.yml', '.yaml'}
+
+
+def _detect_windows_subst_drive():
+    """Detect if gitman is started the from root of a virtual drive."""
+    cwd = os.getcwd()
+    major, minor, *_ = sys.version_info
+    if major == 3 and minor >= 8:
```

Comment:
`sys.version_info >= (3, 8)` should work here.

Reply:
You are right that a lot cleaner.